project1/client/client.cpp

```diff
@@ -158,14 +111,15 @@ void Client::Output() {
 }
 bool Client::FtpShell() {
   ftp_messages::Request r;
+  thread_pool::ThreadPool pool;
 
   while (connected_) {
-    // reset input string
-    std::string input;
-
     // will have to pay attention to formatting here when implementing Output()
```

Comment:
Please remove this comment. I meant to remove it before submitting p1 but a clearly didn't...